lisa/base_tools/uname.py

```diff
@@ -80,6 +80,11 @@ def get_linux_information(
 
         return result
 
+    def get_machine_architecture(self, force_run: bool = False) -> str:
```

Comment:
Can it reuse the arch enum in lscpu?

Reply:
Yes, we can reuse lscpu.get_architecture(). 
However, I don't think this is more widely supported than "uname -m". And the assert message in get_architecture is not clear.